### PR TITLE
Issue3628 rebasedockerfiles

### LINKED ISSUE #3628: Feature Request: rebase and restructure ESS Dockerfiles
### Is your feature request related to a problem? Please describe.

We are running insecure base images.

There are more efficient ways to write these Dockerfiles to generate smaller runtime images.

See a good first step in #3626

### Describe the solution you'd like.

_No response_

### Describe alternatives you've considered

_No response_

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by johnwalicki**
# Pull Request Template

## Description

Rebase all of the Dockerfile.ubi files to ubi9/ubi-minimal:9.1

Fixes #3628

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Review by linggao — DISMISSED** [review]
Hold on until it is fully tested.

**Comment by johnwalicki** [comment]
I was able to build all of these containers, but we should test them.

**Comment by johnwalicki** [comment]
> Hold on until it is fully tested.

This is ready for everyone to test.

**Comment by johnwalicki** [comment]
The build is failing, in a section of e2edev that is NOT relevant to these Dockerfile rebase changes.

File: test/gov/run_kube.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ sleep 2
 if [ $RC -ne 0 ]
 then
 	echo "Try to install microk8s"
-	sudo snap install microk8s --classic --channel=1.23/stable
+	sudo snap install microk8s --classic --channel=1.18/stable
```
Comment (LiilyZhang) [inline test/gov/run_kube.sh:39]: @MaxMcAdam Do you know why we move back the microk8s version in e2edev?

**Review by LiilyZhang — APPROVED** [review]

**Comment by johnwalicki** [comment]
Fantastic news - I'm interested in hearing how the rebase improves Open Horizon security and performance.

**Pull request merged**